tests/conftest.py

```diff
@@ -22,3 +23,22 @@ def httpbin_secure_untrusted(monkeypatch, httpbin_secure):
     """
     monkeypatch.delenv('REQUESTS_CA_BUNDLE')
     return httpbin_secure
+
+
+# pytest-httpbin currently does not support chunked requests:
+# <https://github.com/kevin1024/pytest-httpbin/issues/33>
+# <https://github.com/kevin1024/pytest-httpbin/issues/28>
+@pytest.fixture(scope='session')
+def _httpbin_with_chunked_support_available():
+    try:
+        socket.gethostbyname('pie.dev')
+        return True
+    except OSError:
+        return False
+
+
+@pytest.fixture(scope='function')
+def httpbin_with_chunked_support(_httpbin_with_chunked_support_available):
+    if _httpbin_with_chunked_support_available:
+        return 'http://pie.dev'
+    pytest.skip('pie.dev not resolvable')
```

Comment:
Could we change this so that `pie.dev` is not hard-coded but rather taken from the `HTTPBIN_WITH_CHUNKED_SUPPORT` constant?

Reply:
Sure, on it.

Reply:
Done.